packages/application-shell/src/components/configure-intl-provider/configure-intl-provider.js

```diff
@@ -17,14 +16,19 @@ const hideAppLoader = once(() => {
    *       the app would never show (always show the loading screen)
    */
   if (window.onAppLoaded) window.onAppLoaded();
-});
+};
 
 const ConfigureIntlProvider = props => {
-  hideAppLoader();
+  if (!props.locale) {
+    return null;
```

Comment:
It should be fine to do it like this, because we have the "app loader". Once a locale is available, we remove the app loader (see `useEffect`) and render the rest of the app.